public/js/office.web.js

```diff
@@ -2,69 +2,66 @@ $(() => {
 
   Sentry.init({ dsn: 'https://cd95f03dd404470a8988fb776de774da@sentry.io/1441017' });
 
-  const enterRoom = $('[enter-room]');
+  
   const matrixProfile = new MatrixProfile();
 
   if (matrixProfile.isProfileStored()) {
-    enterInOffice(matrixProfile);
+    initOffice(matrixProfile);
+    initLoggoutButton(matrixProfile);
+    initHeaderName(matrixProfile);
   } else {
     redirectToHome();
   }
 
-  //set user name
-  $("#userName").text("Whats'up " + matrixProfile.userName() + "!");
-
-  const logoutButton = $('#btnLogout');
-
-  logoutButton.on('click', (e) => {
-    const auth2 = gapi.auth2.getAuthInstance();
-    auth2.signOut().then(() => {
-      matrixProfile.terminate();
-      auth2.disconnect();
+  function initHeaderName(matrixProfile){
+    $("#userName").text("Whats'up " + matrixProfile.userName() + "!");  
```

Comment:
In the next steps, we could extract this string message from this module to a module responsible to deal with the strings or apply an I18n approach.